### PR TITLE
adding timing changes to yarpdataplayer

### PR DESCRIPTION
**Pull request by vtikha**
This PR fixes the issues where timestamps were not consistent as reported in #2049 #2052.

### CODE REVIEW

File: src/yarpdataplayer/include/worker.h [hunk 1]
```diff
@@ -130,15 +130,23 @@ class MasterThread : public QObject,  public yarp::os::PeriodicThread
     int                     numThreads;
     double                  timePassed, initTime, virtualTime;
     bool                    stepfromCmd;
+    
+    typedef std::chrono::time_point<std::chrono::steady_clock> Moment;
+    
+    void Initialize();
+    
+    void Tick();
```
Comment (Nicogene) [inline src/yarpdataplayer/include/worker.h]: I personally don't like function names upper case, but it is a personal taste, feel free to ignore it 😉 
Moreover it is not a public interface, then not a big deal.

File: src/yarpdataplayer/include/worker.h [hunk 2]
```diff
@@ -130,15 +130,23 @@ class MasterThread : public QObject,  public yarp::os::PeriodicThread
     int                     numThreads;
     double                  timePassed, initTime, virtualTime;
     bool                    stepfromCmd;
+    
+    typedef std::chrono::time_point<std::chrono::steady_clock> Moment;
```
Comment (Nicogene) [inline src/yarpdataplayer/include/worker.h]: I'd prefer `using Moment = std::chrono::time_point<std::chrono::steady_clock>;`

File: src/yarpdataplayer/include/worker.h (hunk 1 above)
Comment (vtikha) [reply src/yarpdataplayer/include/worker.h]: Correct, changed them, thanks

File: src/yarpdataplayer/include/worker.h (hunk 2 above)
Comment (vtikha) [reply src/yarpdataplayer/include/worker.h]: changed thank you

**Comment by Nicogene** [comment]
Merged, thanks!

**Pull request merged**